log4j.properties

```diff
@@ -0,0 +1,23 @@
+kafka.logs.dir=logs
+
+log4j.rootLogger=INFO, stdout, appender
+
+# Disable excessive reflection warnings - KAFKA-5229
+log4j.logger.org.reflections=ERROR
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.threshold=INFO
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p  %X{dbz.connectorType}|%X{dbz.connectorName}|%X{dbz.connectorContext}  %m   [%c]%n
+
+
+log4j.appender.appender=org.apache.log4j.RollingFileAppender
+log4j.appender.appender.policy.type=SizeBasedTriggeringPolicy
+log4j.appender.appender.policy.size=100MB
+log4j.appender.appender.strategy.type=DefaultRolloverStrategy
+log4j.appender.appender.File=${kafka.logs.dir}/connect-service.log
+log4j.appender.appender.ImmediateFlush=true
+log4j.appender.appender.MaxBackupIndex=10000
+log4j.appender.appender.layout=org.apache.log4j.PatternLayout
+log4j.appender.appender.filePattern=${kafka.logs.dir}/connect-service-%d{yyyy-MM-dd}-%i.log
```

Comment:
As discussed offline, we are only seeing a counter being appended at the end of "\<filename\>.log.\<counter\> ". Since the date is not getting appended to the filename, this config is not working correctly at the moment. We can remove this and add a TODO to have date & time appended to the filename.